src/components/buttonGroup/ButtonGroup.tsx

```diff
@@ -1,22 +1,44 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
+import React, { PureComponent, ReactElement } from 'react';
 import omit from 'lodash.omit';
-import Box, { omitBoxProps } from '../box';
+import Box, { omitBoxProps, pickBoxProps } from '../box';
 import Button from '../button';
 import cx from 'classnames';
 import isComponentOfType from '../utils/is-component-of-type';
 import theme from './theme.css';
+import { BoxProps } from '../box/Box';
+import { BUTTON_LEVELS } from '../button/Button';
 
-/** @type {React.ComponentType<any>} */
-class ButtonGroup extends PureComponent {
-  handleChange = (value, event) => {
+interface ButtonGroupProps extends Omit<BoxProps, 'size' | 'onChange' | 'ref'> {
```

Comment:
I think this added all the props of `Box` in the documentation, if you only need size, onChange and ref doesn't this have to be `Pick` instead of `Omit`?